_posts/2021-01-06-introducing-artsy-engineering-radio.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: epic
+title: "Introducing Artsy Engineering Radio"
+date: 2021-01-06
+categories: [podcast, community, learning]
+author: [steve-hicks, jonallured]
+---
+
+Available now [on iTunes][itunes] and hopefully your preferred podcast provider 🤞 is Artsy Engineering Radio!
```

Comment:
Not feeling great about "hopefully your preferred podcast provider 🤞" because it feels unclear whether I mean "hopefully you like itunes" instead of "hopefully we get those other sources working". Open to suggestions.

Reply:
Maybe something like...
```suggestion
Available now [on iTunes][itunes] (and coming elsewhere soon) is Artsy Engineering Radio!
```

Reply:
Should this be "Apple Podcasts"? On Mac that seems to be what it would be, though on Windows it's still iTunes (see [here](https://www.apple.com/itunes/)).

Re: wording, how about "Available now on Apple Podcasts/iTunes (and soon to be available on Spotify, Stitcher, and Google podcasts): Artsy Engineering Radio!"